packages/calypso-e2e/src/lib/flows/start-site-flow.ts

```diff
@@ -21,6 +21,8 @@ const selectors = {
 
 	// Themes
 	individualThemeContainer: ( name: string ) => `.design-button-container:has-text("${ name }")`,
+	themeContinueButton: '.design-preview__sidebar-action-buttons:has-text("Continue")',
+	themeContinueButtonSmall: '.step-container__navigation:has-text("Continue")',
```

Comment:
**issue:blocking** - because the distinction is between FSE and non-FSE themes, this selector is not actually going to work if we opt to test a theme that is not FSE. That's because for such themes, the button to confirm theme selection reads "Start with ${themeName}` instead.

I should have been clearer that we do not need to worry about mobile viewports for the time being.

Reply:
**issue:continuing** - Furthermore with Playwright adding support for obtaining locators by one of the many a11y attributes, it would be preferable to begin deprecating the use of selector objects as time goes on.

An example of what Playwright can do:

```typescript
await this.page.getByRole( 'button', { name: "Continue" } ).click();
```